homu/server.py

```diff
@@ -406,9 +406,11 @@ def report_build_res(succ, url, builder, state, logger, repo_cfg):
             utils.github_create_status(state.get_repo(), state.head_sha, 'success', url, desc, context='homu')
 
             urls = ', '.join('[{}]({})'.format(builder, x['url']) for builder, x in sorted(state.build_res.items()))
-            state.add_comment(':sunny: {} - {}'.format(desc, urls))
+            test_comment = ':sunny: {} - {}'.format(desc, urls)
 
             if state.approved_by and not state.try_:
+                comment = test_comment + '\n' + 'Approved by={}\nPushing {} to {}...'.format(state.approved_by, state.merge_sha, state.base_ref)
```

Comment:
nit: Approved by uses a ```:``` instead of ```=``` in [main.py](https://github.com/barosl/homu/blob/master/homu/main.py#L481).